Formula/parity.rb

```diff
@@ -2,22 +2,22 @@
 
 class Parity < Formula
   homepage "https://github.com/thoughtbot/parity"
-  sha256 "d56827ce379958b5abe386f6afff3b012275f43a6d982f524c54bb8a790cee20"
-  url "https://github.com/thoughtbot/parity/releases/download/v0.9.2/parity-0.9.2-osx.tar.gz"
-  version "0.9.2.1"
+  sha256 "438b15beb0f0e0fd8da1aee5cc47ca57ff03c138091100d30fe5ae90a0f22c22"
+  url "https://github.com/thoughtbot/parity/releases/download/0.9.3/parity-0.9.3-osx.tar.gz"
+  version "0.9.3"
 
   depends_on "git"
   depends_on "heroku-toolbelt"
   depends_on "postgres"
 
   def install
-    prefix.install "lib" => "lib"
+    lib.install Dir["lib/*"]
 
     bin.install "bin/development", "bin/staging", "bin/production"
   end
 
   devel do
-    url "https://github.com/thoughtbot/parity/releases/download/v0.9.3.beta/parity-0.9.3.beta-osx.tar.gz"
+    url "https://github.com/thoughtbot/parity/releases/download/v0.9.3.beta/parity-0.9.3.beta2-osx.tar.gz"
```

Comment:
UPDATE: I created a development release with a fixed name so we don't have to keep updating this formula line over time as we publish new development version.